### ISSUE TITLE
Missing check for duplicate classes in different bundle blocks

### ISSUE THREAD

**Issue by objeck** [objeck/objeck-lang#37]
The following code should generate a compiler error.

```
bundle Foo {
	class Test {
		function : Main(args : String[]) ~ Nil {
		}
	}

	class Bar {
		New() {}
	}
}

bundle Foo {
	class Bar {
		New() {}
	}
}
```

**Comment by objeck** [comment]
fixed in 5.9.7

**Comment by objeck** [comment]
fixed in 5.9.7
